internal/web/results_test.go

```diff
@@ -50,9 +50,12 @@ func TestResultsODML(t *testing.T) {
 	body := []byte("{}")
 	router := mux.NewRouter()
 	router.HandleFunc("/results/{validator}/{user}/{repo}/{id}", Results).Methods("GET")
-	r, _ := http.NewRequest("GET", filepath.Join("/results/odml", username, "/", reponame, "/", id), bytes.NewReader(body))
+	r, _ := http.NewRequest("GET", filepath.Join("/results/odml", username, reponame, id), bytes.NewReader(body))
 	w := httptest.NewRecorder()
-	resultfldr, _ := ioutil.TempDir("", "results")
```

Comment:
Had to change the way the temp folders are created, because `resultsHistory` function expects a `results` folder to be somewhere in the path